assets/queries/dockerfile/secrets_env/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Secrets_env",
+  "queryName": "Secrets Env",
```

Comment:
The query name should be clarifying of the problem in the resource, in this case, that there are secrets stored in the Dockerfile